core/internal/gethwrappers/go_generate_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const compileCommand = "../scripts/native_solc_compile_all"
+const compileCommand = "../../../evm-contracts/scripts/native_solc_compile_all"
```

Comment:
ah saw that locally as well

Reply:
Not really sure how this ever worked tbh